roles/rsyslog_server/defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+firewall_allowed_tcp_ports:                 # list of open ports on iCAT server
+  - "22"                  # SSH.
+  - "{{ rsyslog_port | default('514') }}"  # defined in inventory
+
+# If the sister role of 'rsyslog_client' will also be run on the, then it is
```

Comment:
# If the sister role of 'rsyslog_client' will also be run on the **<something is missing here>**, then it is